pkg/webhook/webhook.go

```diff
@@ -25,12 +25,14 @@ import (
 
 	"sigs.k8s.io/kubefed/pkg/controller/webhook"
 	"sigs.k8s.io/kubefed/pkg/controller/webhook/federatedtypeconfig"
+	"sigs.k8s.io/kubefed/pkg/controller/webhook/kubefedconfig"
 )
 
 func NewWebhookCommand(stopChan <-chan struct{}) *cobra.Command {
 	admissionHooks := []apiserver.AdmissionHook{
 		&federatedtypeconfig.FederatedTypeConfigValidationHook{},
 		&webhook.KubeFedClusterValidationHook{},
+		&kubefedconfig.KubeFedConfigValidationHook{},
```

Comment:
@font Why are the webhooks for federatedtypeconfig and kubefedconfig in a separate package but that isn't the case for kubefedcluster?

Reply:
It's because `kubefedcluster` validations are still WIP and will also be moved to its own separate package when it's complete.